### PR TITLE
LPS-155271 | Added automated tests 

### PR DESCRIPTION
**Pull request by andreluis2022**
My Ticket:
https://issues.liferay.com/browse/LPS-155271

Description:
Create Automated tests

Minified#VerifyErroWhenJQueryEnabled

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase [hunk 1]
```diff
@@ -31,7 +31,19 @@ definition {
...
+		portlet = "System Settings");
+
+	SystemSettings.gotoConfiguration(
+		configurationCategory = "Third Party",
+		configurationName = "JQuery",
+		configurationScope = "System Scope");
+
+	FormFields.enableCheckbox(fieldName = "Enable JQuery");
+
+	SystemSettings.saveConfiguration();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase:46]: Wrap the poshi steps with tasks( ), please see https://github.com/mateusralv/liferay-portal/pull/41/files#diff-3fdc3c718f6e440ff2b4cee5b74080c0c3ecc97076cd214ea19df03cfdd9f16dR101-R118

Ask @abhnerramos if need help with it.

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

Run `ci:test:frontend:nocompile` after updating the test with tasks. Please ping @eltonccdantas or @deborasoliveira to review and they can `ci:forward`

**Comment by andreluis2022** [comment]
ci:test:frontend:nocompile

**Comment by andreluis2022** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase (hunk 1 above)
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase]: ok

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
Hey @andreluis2022 , please don't forget to: 

- Check `source-format`  before sending a pull request.
- Attach the poshi report `index.html`  to your PR

Your test  `Minified#VerifyErroWhenJQueryEnabled` has failed on `Click( key_category = "${category}", key_portlet = "${portlet}", locator1 = "ApplicationsMenu#PORTLET");` , Please check the XPath  and variables needed. 

Feel free to look for a solution for it tomorrow morning... If you still need help, we can work on a solution tomorrow afternoon.

**Comment by john-co** [comment]
Hey @andreluis2022 , it looks like the test ClientExtensionUsingConfigFile#OsgiClientExtensionWorkspaceLabelIsPresent was added to this PR in the middle of reviewing Minified#VerifyErroWhenJQueryEnabled. Do not do this as this is confusing for everyone to keep track of the work in each ticket. Please create a separate PR for each ticket.

**Comment by andreluis2022** [comment]
> Ei@andreluis2022, parece que o teste ClientExtensionUsingConfigFile#OsgiClientExtensionWorkspaceLabelIsPresent foi adicionado a este PR no meio da revisão de Minified#VerifyErroWhenJQueryEnabled. Não faça isso, pois isso é confuso para todos acompanharem o trabalho em cada tíquete. Por favor, crie um PR separado para cada ticket.



> Ei@andreluis2022, não se esqueça de:
> 
> * Verifique `source-format`  antes de enviar uma solicitação pull.
> * Anexe o relatório poshi `index.html`  ao seu PR
> 
> Seu teste `Minified#VerifyErroWhenJQueryEnabled`falhou em `Click( key_category = "${category}", key_portlet = "${portlet}", locator1 = "ApplicationsMenu#PORTLET");`, verifique o XPath e as variáveis ​​necessárias.
> 
> Sinta-se à vontade para procurar uma solução para isso amanhã de manhã... Se ainda precisar de ajuda, podemos trabalhar em uma solução amanhã à tarde.

ok I will review again the code and the source-format and update the pr, thanks for the help @eltonccdantas @john-co

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by deborasoliveira** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hey @andreluis2022 could you run SF locally and update this PR?

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
The test failed in the CI, but the failure does not seem related to your test, Rou run the suite once again

**Comment by mateusralv** [comment]
Hey @andreluis2022 The test is still[ failing on CI](https://testray.liferay.com/home/-/testray/case_results/2196730165?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2196729890%26testrayRunId%3D2196729920%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)... could take a look?

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
continue: https://github.com/liferay-frontend/liferay-portal/pull/3069

**Pull request closed without merging**